.readthedocs.yaml

```diff
@@ -1,4 +1,15 @@
 version: 2
-mkdocs:
-  configuration: mkdocs.yml
-  fail_on_warning: false
+
+formats:
+  - pdf
+  - epub
+
+python:
+  version: "3.6"
+  install:
+    - requirements: requirements.txt
```

Comment:
Oof, another reference to this file.  This reminds me of the comment I made a while back re: We need to check/test this at the CI VM-image build level.  Otherwise what we test != what we ship.   I'll open a Jira card...okay it's done.